jest.config.js

```diff
@@ -61,6 +61,7 @@ module.exports = {
     'app/**/*.{js,jsx,ts,tsx}',
     'lib/**/*.{js,jsx,ts,tsx}',
     'addons/**/*.{js,jsx,ts,tsx}',
+    '!**/*.stories.{js,jsx,ts,tsx}',
```

Comment:
I've tried to put this pattern in `coveragePathIgnorePatterns`, but it fails with `SyntaxError: Invalid regular expression: /**/*.stories.{js,jsx,ts,tsx}/: Nothing to repeat`.

Reply:
@mbelsky if it helps, i think `**` is a glob not a regex

Reply:
Fixed, thanks